apb.yml

```diff
@@ -62,3 +64,5 @@ plans:
       longDescription: This plan provides a single non-HA PostgreSQL server with persistent storage
       cost: $0.00
     parameters: *_p
+    updates_to:
```

Comment:
Out of curiosity, can you update to yourself (ie. `prod` 9.4 to `prod` 9.6)?

Reply:
Yes. As long as _something_ changes you can update. That can be plan, version, or plan and version. If nothing changes the UI won't let you submit. If you did submit an update and nothing changed (via the API directly for instance) it would basically be a no op.